test/test_sax.rb

```diff
@@ -182,6 +184,22 @@ def test_with_default_entity
             sax.parse
           end
         end
+
+        def test_entity_expansion_text_limit
+          source = <<-XML
+<!DOCTYPE member [
+  <!ENTITY a "&b;&b;&b;">
+  <!ENTITY b "&c;&d;&e;">
+  <!ENTITY c "xxxxxxxxxx">
+  <!ENTITY d "yyyyyyyyyy">
+  <!ENTITY e "zzzzzzzzzz">
+]>
+<member>&a;</member>
+          XML
+
+          REXML::Security.entity_expansion_text_limit = 90
+          REXML::Parsers::SAX2Parser.new(source).parse
```

Comment:
Can we check the text size here something like the last patch in https://github.com/ruby/rexml/pull/194#issuecomment-2279113530 ?

Reply:
OK, I see.